fabtests/prov/lpp/src/rcq_data.c

```diff
@@ -16,7 +16,7 @@ int run_fi_tsenddata(struct rank_info *ri){
 	size_t ndata = sizeof(rcq_data)/sizeof(*rcq_data);
 	size_t nbufflens = sizeof(buff_lens)/sizeof(*buff_lens);
 
-	if (ndata == nbufflens)
+	if (ndata != nbufflens)
 		return -EINVAL;
```

Comment:
Why having this check at all? The two values are always equal based on how the two arrays are initialized.

Reply:
That was an assert probably left there as the code evolved.
I have reworked the code to use buffers with static size and removed to check.

Reply:
PR updated.